code/game/objects/explosion.dm

```diff
@@ -60,14 +60,25 @@
 					var/dist = get_dist(M_turf, epicenter)
 					// If inside the blast radius + world.view - 2
 					if(dist <= round(max_range + world.view - 2, 1))
-						M.playsound_local(epicenter, get_sfx("explosion"), 100, 1, frequency, falloff = 5) // get_sfx() is so that everyone gets the same sound
+						if(devastation_range > 0)
+							M.playsound_local(epicenter, get_sfx("explosion"), 100, 1, frequency, falloff = 5) // get_sfx() is so that everyone gets the same sound
+						else
+							M.playsound_local(epicenter, get_sfx("explosion_small"), 100, 1, frequency, falloff = 5)
+						spawn()
+							shake_camera(M, 10, 2)

```

Comment:
This already has spawn(1) inbuilt (though I dont even like the method they do it with really)